drivers/display/display_mcux_elcdif.c

```diff
@@ -47,44 +51,44 @@ static int mcux_elcdif_write(const struct device *dev, const uint16_t x,
 			     const void *buf)
 {
 	const struct mcux_elcdif_config *config = dev->config;
-	struct mcux_elcdif_data *data = dev->data;
+	struct mcux_elcdif_data *dev_data = dev->data;
```

Comment:
PR mentions there is a clash, where?

Reply:
Similar to the change in PR https://github.com/zephyrproject-rtos/zephyr/pull/42077/files